### ISSUE TITLE
Saved 'Untitled {type} block' successfully

### ISSUE THREAD

**Issue by robbieaverill** [dnadesign/silverstripe-elemental#600]
**4.0.x-dev elemental with SS 4.3.x-dev**

If you save a block without a title, you get this toast message:

![image](https://user-images.githubusercontent.com/5170590/52529659-f67dbd00-2d06-11e9-97db-acc7994cf435.png)

I guess it should say "Saved untitled block successfully" with no capitalisation or placeholder if there's no title set.

**Comment by robbieaverill** [comment]
>I guess it should say "Saved untitled block successfully" with no capitalisation or placeholder if there's no title set.

Actually this message is already for when there's no title set, it's just not pulling in the block type correctly:

```js
    const noTitle = i18n.inject(
      i18n._t(
        'ElementHeader.NOTITLE',
        'Untitled {type} block'
      ),
      { type: element.BlockSchema.type }
    );
```

**Comment by robbieaverill** [comment]
PR at #601

**Comment by ScopeyNZ** [comment]
Fixed in #601
